crates/ty_python_semantic/resources/mdtest/cycle.md

```diff
@@ -31,3 +31,18 @@ p = Point()
 reveal_type(p.x)  # revealed: Unknown | int
 reveal_type(p.y)  # revealed: Unknown | int
 ```
+
+## Referencing current value
+
+A variable or attribute may reference its current value as part of an assignment.
+
+```py
+class C:
+    def __init__(self):
+        self.a = 0
+
+    def incr(self):
+        self.a = self.a + 1
+        # revealed: Unknown | int
+        reveal_type(self.a)
+```
```

Comment:
This can be removed. We have more or less the same test in `attributes.md` (search for "`class Counter`").